models/jira.yml

```diff
@@ -34,6 +34,41 @@ models:
         description: The story point estimate that an issue has on a given day.  
         # other fields included in var(issue_history_columns)
 
+  - name: jira__timestamp_issue_field_history
+    description: >
+      Each record represents a time period when an issue had specific field values. This is an SCD Type 2 table
+      that tracks field changes at the timestamp level, showing the complete field state during each validity period.
+      Each record contains valid_from/valid_until timestamps and an is_current_record flag.
+    columns:
+      - name: valid_from
+        description: Timestamp when this field state became effective.
+      - name: valid_until
+        description: Timestamp when this field state ended (next change or current time).
+      - name: updated_at_week
+        description: The start date of the week corresponding to the update timestamp.
+      - name: issue_id
+        description: Foreign key referencing the ID of the `issue` with these values.
+      - name: status_id
+        description: The ID of the status that an issue has during this period.
+      - name: status
```

Comment:
`status_category` needs to be documented for this table as it's a default field.

Reply:
Added